Sources/SwiftProtobuf/_MessageStorage.swift

```diff
@@ -540,6 +547,158 @@ extension _MessageStorage {
     // - Enums
 }
 
+// MARK: - Field mutators used for parsing and reflection APIs
+
+// Unlike the above APIs, these only take a `FieldLayout` as an argument. These are used when
+// parsing messages and in reflection APIs, where we don't already know at generation time (as we
```

Comment:
Not sure I follow this.  We always know all this at "generation" time, is this about not knowing it within say the generic table code that parses binaries?  i.e. - it will have to be looked up at _runtime_ because we don't _generate_ things specifically for serialization?

Reply:
Right, this was poor phrasing on my part. Updated it.